src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--name-of-the-day-mon">
```

Comment:
in order to make your project better you can update your structure;
it would be better to create a folder 'utils ' inside 'styles' folder.
put your 'variable.scss' inside 'utils' folder (in the future on the project you can have additional css files, so its good to implement some basic structure at the beginning)